file2.txt

```diff
@@ -1,2 +1,4 @@
-bye
```

Comment:
Hey Francisco, please delete this line because is creating a conflict. Send it again and I'll merge! Thanks!

Reply:
OK! I will delete the conflicted line.
You closed the pull request though... I will create a new one!